core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentRefreshInfo.java

```diff
@@ -21,12 +21,17 @@
 
 public class SegmentRefreshInfo implements Serializable {
 
-  private Long segmentUpdatedTimestamp;
+  private Long segmentUpdatedTimestamp = 0L;
   private Integer countOfFileInSegment;
+  private Long segmentFileTimestamp = 0L;
 
-  public SegmentRefreshInfo(Long segmentUpdatedTimestamp, Integer countOfFileInSegment) {
-    this.segmentUpdatedTimestamp = segmentUpdatedTimestamp;
+  public SegmentRefreshInfo(Long segmentUpdatedTimestamp, Integer countOfFileInSegment,
+      Long segmentFileTimestamp) {
+    if (segmentUpdatedTimestamp != null) {
```

Comment:
In case of update scenario, If we fill `segmentUpdatedTimestamp` as `segmentFileTimestamp`, will there be a problem ? I think no need to have `segmentFileTimestamp` variable. We can just  use `segmentUpdatedTimestamp` and fill this based on segment modified time

Reply:
please analyze if this can work

Reply:
1. I didnt understand why we fill the update time as segment time? Can you please tell? you mean if developer sets by mistake or something else?
2. Are you saying to completely remove the segmentUpdateTimeStamp during refresh?

Reply:
@ajantha-bhat Why will we segmentUpdatedTimestamp as segmentFileTimestamp??

Reply:
@kunal642 , @akashrn5 : Isn't the segmentUpdatedTimestamp will be same as latest modified segment time. which is segmentFileTimestamp ?

But old store, which doesn't have segmentFile update may not work If we do that.  We cannot do this now.

Reply:
@ajantha-bhat you are right, first thing is old store case and `segmentUpdatedTimestamp` and `segmentFileStamp` wont be same because `segmentFileStamp` is file's last modified time, and `segmentUpdatedTimestamp` we actually set in the table status after update. So we can keep both for old and new store scenarios.

Reply:
@ajantha-bhat We already have a check for segmentFile existance to handle compatibility.
And incase of multiple drivers the driver which has old information this code will check the latest timestamp of the file with the updateVo to check whether any other driver has updated or not.

Reply:
ok